includes/admin/class-wp-job-manager-settings.php

```diff
@@ -823,6 +826,18 @@ protected function input_multi_enable_expand( $option, $attributes, $values, $pl
 		$enable_option['name']       = $option['name'] . '[' . $enable_option['name'] . ']';
 		$enable_option['type']       = 'checkbox';
 		$enable_option['attributes'] = [ 'class="sub-settings-expander"' ];
+
+		if ( isset( $enable_option['force_value'] ) && is_bool( $enable_option['force_value'] ) ) {
+			if ( true === $enable_option['force_value'] ) {
```

Comment:
Could this just be `if ( $enable_option['force_value'] ) { ... } else { ... }`? We've already checked for `is_bool` so I'm not sure we need to be so specific with our if/else here.

Reply:
Good shout 44b93fd038bddada0d5ba2416b0d8bd3e92bb654